nft_ingester/src/program_transformers/bubblegum/db.rs

```diff
@@ -532,70 +532,46 @@ pub async fn upsert_asset_creators<T>(
 where
     T: ConnectionTrait + TransactionTrait,
 {
-    // Vec to hold base creator information.
-    let mut db_creators = Vec::with_capacity(creators.len());
-
-    if creators.is_empty() {
-        // Bubblegum supports empty creator array.  In this case insert an empty Vec
-        // for the creator.
-        db_creators.push(asset_creators::ActiveModel {
+    let db_creators = creators
+        .iter()
+        .enumerate()
+        .map(|(i, creator)| asset_creators::ActiveModel {
             asset_id: Set(id.clone()),
-            position: Set(0),
```

Comment:
I think we still need this Bubblegum code that inserts the "empty" creator since unlike Token Metadata, Bubblegum allows an empty creator array.

Reply:
Fixed. Good catch.